docs/developer-docs/functionality/http_requests/http_requests-how-it-works.md

```diff
@@ -116,15 +120,11 @@ The following &ldquo;recipe&rdquo; gives you a blueprint of how to best tackle t
 * Determine the maximum response size of the server's response for this type of request to populate the `max_response_bytes` parameter with it. This often works well for API calls and is important to not overcharge the requesting canister in terms of cycles. The `HEAD` request type can be used to do this if responses change dynamically in response size considerably.
 * Implement the request and try it out in the local SDK environment. However, note that the behaviour of the local environment does not reflect that of the IC as there is only one replica in the local environment and $n$, e.g., $n=13$, replicas on an IC subnet.
 
-:::note
-**Pro tip:** Do not forget to consider response headers when identifying the variable parts of your response because headers sometimes contain variable items such as timestamps which may prevent the responses to go through consensus.
-:::
+:::tip Pro tip
 
-### Debugging
+Do not forget to consider response headers when identifying the variable parts of your response because headers sometimes contain variable items such as timestamps which may prevent the responses to go through consensus.
```

Comment:
```suggestion
Do not forget to consider response headers when identifying the variable parts of your response because headers sometimes contain variable items such as timestamps which may prevent the responses from passing through consensus.
```